notifications/05-10-23-0900.md

```diff
@@ -0,0 +1,77 @@
+**What is happening?**
+
+CHG0048445 - MCS SILVER - 2023 Q2 Patching - Manual Image Prune, Hard Prune of Image registry, ETCD Defrag and Trident Upgrade
+
+CHG0048444 - MCS SILVER - 2023 Q2 Patching - Upgrade to OCP 4.12.14
+
+CHG0048466 - MCS GOLD - 2023 Q2 Patching - Manual Image Prune, Hard Prune of Image registry, ETCD Defrag and Trident Upgrade
+
+CHG0048467 - MCS GOLD - 2023 Q2 Patching - Upgrade to OCP 4.12.14
+
+XXXX - MCS GOLDDR - 2023 Q2 Patching - Manual Image Prune, Hard Prune of Image registry, ETCD Defrag and Trident Upgrade
+
+XXXX - MCS GOLDDR - 2023 Q2 Patching - Upgrade to OCP 4.12.14
+
+These six changes represent the work involved to upgrade the production clusters from Openshift 4.11 to 4.12.
+
+**When?**
+
+CHG0048445 - MCS SILVER - 2023 Q2 Patching - Manual Image Prune, Hard Prune of Image registry, ETCD Defrag and Trident Upgrade will commence at 0600 on Sunday May 28th and run as late as 1200.
+
+CHG0048444 - MCS SILVER - 2023 Q2 Patching - Upgrade to OCP 4.12.14 will commence at 9:00am on Monday May 28th. Upgrade efforts will be allowed to proceed between 9:00am and 5:00pm, and be paused after-hours to ensure developers are present during node drains to look after their apps if needed.
+
+CHG0048466 - MCS GOLD - 2023 Q2 Patching - Manual Image Prune, Hard Prune of Image registry, ETCD Defrag and Trident Upgrade will commence at 0600 on Sunday June 11th and run as late as 1200.
+
+CHG0048467 - MCS GOLD - 2023 Q2 Patching - Upgrade to OCP 4.12.14 will commence at 9:00am on  Monday June 12th. Upgrade efforts will be allowed to proceed between 9:00am and 5:00pm, and be paused after-hours to ensure developers are present during node drains to look after their apps if needed.
+
+XXXX - MCS GOLDDR - 2023 Q2 Patching - Manual Image Prune, Hard Prune of Image registry, ETCD Defrag and Trident Upgrade will commence at 0600 on Sunday June 18th and run as late as 1200.
+
+XXXX - MCS GOLDDR - 2023 Q2 Patching - Upgrade to OCP 4.12.14 will commence at 9:00am on Monday June 19th. Upgrade efforts will be allowed to proceed between 9:00am and 5:00pm, and be paused after-hours to ensure developers are present during node drains to look after their apps if needed.
+
+**Will there be an impact on the Platform apps?**
+
+CHG0048445 will involve removing some old images and defragmenting the ETCD database. Image tags that are not currently in use by a pod or deployment and are older than 14 days will be pruned from the registry completely. Defragmenting ETCD could cause some latency and slowness but isn't expected to cause any breakage as each pod will be done individually with time to recover between operations. The hard prune will put the registry in read-only mode briefly while the pruner removes any orphaned images. The Trident upgrade may cause a brief window where new storage cannot be allocated, expected to be under 5 minutes. It will not impact existing storage.
```

Comment:
```suggestion
CHG0048445 will involve orphaned image data cleanup, defragmenting the ETCD database, and upgrading the storage provisioner tool Trident. Defragmenting ETCD could cause some latency and slowness but isn't expected to cause any breakage as each pod will be done individually with time to recover between operations. The hard prune will put the registry in read-only mode briefly while the pruner removes any orphaned images. The Trident upgrade may cause a brief window where new storage cannot be allocated, expected to be under 5 minutes. It will not impact existing storage.
```